### ISSUE TITLE
RegularFaceRegularizer: Move cos.clone() inside of with torch.no_grad()

### ISSUE THREAD

**Issue by KevinMusgrave** [KevinMusgrave/pytorch-metric-learning#219]
No need to clone it with ```requires_grad = True```
